packages/devtools_app/lib/src/shared/diagnostics/generic_instance_reference.dart

```diff
@@ -36,3 +39,16 @@ class GenericInstanceRef {
 
   final IsolateRef? isolateRef;
 }
+
+enum ExpandType {
```

Comment:
I think we can name this a bit better. Perhaps, ObjectReferencesChildType or similar?

Reply:
RefNodeType ?